### PR TITLE
Ember-Engines load order patch

### PR DESCRIPTION
**Pull request by XaserAcheron**
I noticed that `ember-electron` is _almost_ compatible with [engines](https://github.com/ember-engines/ember-engines) out of the box. The one troublesome spot is that `ember-engines` does some extra stuff in its `postprocessTree` hook (namely, generating an asset manifest) that relies on the Ember app existing at the tree root. Unfortunately, `ember-electron` happens to run first (thanks, alphabet!) and put the app into an `ember` subdirectory, and causes the manifest generation step to go awry.

Short version: this PR ensures that `ember-engines` does its thing first. Easy one-liner.

I do wonder if this is something that could be handled in `ember-asset-loader` long-term, but for now, this ought to make things a bit saner.

### CODE REVIEW

**Comment by felixrieseberg** [comment]
This is a good PR. Thank you ❤️

**Comment by XaserAcheron** [comment]
Glad to help! Thanks for making this so surprisingly easy to set up. I've suddenly a working desktop app now. O_O
